README.md

```diff
@@ -21,25 +21,57 @@ A difference to existing projects like OpenBCI is that it will not be necessary
         
   <tr>
   <td><img src = "/docs/images/start_screen_1.png" height = "500" width="250"></td>
-
-  <td><img src = "/docs/images/start_screen_2.png" height = "500" width="250"></td>  
-
-  <td><img src = "/docs/images/start_screen_3.png" height = "500" width="250"></td>
-  </tr>
-
-  <tr>
   <td><img src = "/docs/images/Demo.gif" height = "500" width="250"></td>
   <td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
-  <td><img src = "/docs/images/nav_menu.png" height = "500" width="250"></td>
+  </tr>
+  
+  <tr>
+  <td><img src = "/docs/images/focus.gif" height = "500" width="250"></td>
+  <td><img src = "/docs/images/memory_graph_sc.jpeg" height = "500" width="250"></td>
+  <td><img src = "/docs/images/pinlayout.jpeg" height = "500" width="250"></td>
   </tr>
 
   <tr>
+  <td><img src = "/docs/images/nav_menu.png" height = "500" width="250"></td>
   <td><img src = "/docs/images/about_screen.png" height = "500" width="250"></td>
-  <td><img src = "/docs/images/settings_screen.png" height = "500" width="250"></td>
+  <td><img src = "/docs/images/settings_screen.jpeg" height = "500" width="250"></td>
   </tr>
 
   </table>
   
+## Features
+
+### Launcher Screen
+
+<td><img src = "/docs/images/home_screen.png" height = "500" width="250"></td>
+
+The four major program modes: Focus, Relax, Memory Graph, Meditation.
+Each program mode is responsible for specific activities and games with your brain activity.
+
+### Focus Program Mode
+
+<td><img src = "/docs/images/focus_mode.png" height = "500" width="250"></td>
+
+The Focus program mode helps the user to increase focus and concentration power through playing games.
```

Comment:
Focus program mode helps user increase focus and concentration power by playing games.